redaxo/src/addons/be_style/boot.php

```diff
@@ -11,6 +11,9 @@
  * @package redaxo5
  */
 
+/** @var rex_addon $myAddon */
+$myAddon = $this;
```

Comment:
test, da man `$this` nicht via `@var` typehinten kann

Reply:
phpstorm versteht das 😉 
funktioniert hier ja auch https://github.com/redaxo/redaxo/blob/008d9847ee08113447c22c1fe1fbdf7814ec6a69/redaxo/src/addons/project/boot.php#L3-L5

Reply:
ich hatte lokal getestet und dort ging es nicht (zumindset nicht in den closures hier im file)